src/components/Editor/ConditionalPanel.js

```diff
@@ -60,10 +62,15 @@ export class ConditionalPanel extends PureComponent<Props> {
   };
 
   setBreakpoint(condition: string) {
-    const { selectedLocation, line } = this.props;
+    const { selectedLocation, location } = this.props;
+    if (!location) {
+      return;
+    }
+
+    const { line, column } = location;
     const sourceId = selectedLocation ? selectedLocation.sourceId : "";
-    const location = { sourceId, line };
-    return this.props.setBreakpointCondition(location, { condition });
+    const loc = { sourceId, line, column };
```

Comment:
ooh, this is a nice opportunity to set `sourceId` in the conditional panel location.